CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 <!--
 Contains editorialized release notes. Raw release notes should go into `RELEASE-NOTES.txt`.
 -->
+## 19.8
+Experience seamless updates with our latest release! Now enjoy background updates for your order list and dashboard analytics cards. We've enhanced the dynamic dashboard to display unavailable analytics views when not logged into WPCom. Plus, we've fixed large text support in the Bar Code Scanner for better accessibility. Update now for a smoother experience!
```

Comment:
Bar code feels more natural as a single word to me: Barcode Scanner

Reply:
Thanks, for the suggestion! a5ac612d6d7ae923a9ff7e5640744c945b980d60